src/styles/pages/main/main.scss

```diff
@@ -0,0 +1,377 @@
+        max-width: 1170px;
+        margin: 0 auto;
+        padding: 0 20px;
+    }
+
+    &__wrapper {
+        text-align: center;
+    }
+
+    &__title {
+        color: white;
+        font-size: 30px;
+        padding: 50px 0;
+        font-family: "Dancing Script", cursive;
+    }
+
+    &__content {
+        color: white;
+        font-family: 'Open Sans', sans-serif;
+        font-size: 20px;
+        padding: 30px 0;
+        word-spacing: 10px;
+    }
+}
+
+.specificity {
+    background-color: #1c1b1e;
+
+    .container {
+        max-width: 1170px;
+        margin: 0 auto;
+        padding: 50px 0;
+    }
+
+    &__wrapper {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+    }
+
+    &__content {
+        max-width: 40%;
+        text-align: center;
+        padding-right: 20px;
+    }
+    
+    &__title {
+        color: white;
+        font-size: 20px;
+
+        h3 {
+            font-size: 35px;
+            font-family: "Dancing Script", cursive;
+        }
+
+        h2 {
+            font-size: 35px;
+            padding: 10px 0;
+            font-family: "Dancing Script", cursive;
+        }
+    }
+
+    &__text {
+        color: white;
+        font-family: 'Open Sans', sans-serif;
+        font-size: 20px;
+        padding: 30px 0;
+        word-spacing: 10px;
+    }
+
+    &__image {
+        max-width: 500px;
+        max-height: 500px;
+
+        img {
+            width: 100%;
+        }
+    }
+}    
+
+.contacts {
+    background-color: #1c1b1e;
+
+    .container {
+        max-width: 1170px;
+        margin: 0 auto;
+        padding: 20px;
+    }
+
+    &__wrapper {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+    }
+
+    &__image {
+        max-width: 500px;
+        max-height: 500px;
+
+        img {
+            width: 100%;
+        }
+    }
+
+    &__content {
+        max-width: 40%;
+        text-align: center;
+        padding-right: 20px;
+    }
+
+    &__title {
+        color: white;
+        font-size: 30px;
+        font-family: "Dancing Script", cursive;
+    }
+
+    &__text {
+        color: white;
+        font-family: 'Open Sans', sans-serif;
+        font-size: 20px;
+        padding: 30px 0;
+        word-spacing: 10px;
+    }
+}
+
+.staff {
+    background-color: #1c1b1e;
+
+    .container {
+        max-width: 1170px;
+        margin: 0 auto;
+        padding: 20px;
+    }
+
+    &__wrapper {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+    }
+
+    &__content {
+        max-width: 50%;
+        text-align: center;
+        padding-right: 20px;
+    }
+
+    &__title {
+        color: white;
+        font-size: 20px;
+
+        h3 {
+            font-size: 35px;
+            font-family: "Dancing Script", cursive;
+        }
+
+        h2 {
+            padding: 10px 0;
+            font-family: "Dancing Script", cursive;
+        }
```

Comment:
the same as above

(you had better wrap these tags up and add classes
and rewrite this part of code )